pages/dkp/kommander/2.1/projects/project-deployments/continuous-delivery/index.md

```diff
@@ -39,11 +37,22 @@ The following table describes the fields required for each authentication method
 
 <!-- Refer https://fluxcd.io/docs/components/source/gitrepositories/#spec-examples for flux examples (not everything in there is supported) -->
 
+If you are using GitOps by using a GitHub repo as your source, you can create your secret with a [personal access token][github-token].
+Then, in the DKP UI, in your project, create a Secret, with a key:value pair of `password`: `<your-token-created-on-github>`.
+If you are using a GitHub personal access token, you do not need to have a key:value pair of `username`: `<your-github-username>`.
+
+If you are using a secret with your GitHub username and your password, you will need one secret created in the DKP UI, with key:value pairs of `username`: `<your-github-username>` and `password`: `<your-github-password>`.
+**Note:** if you have multi-factor authentication turned on in your GitHub account, using your GitHub username and password will not work.
+
+<p class="message--note"><strong>NOTE: </strong>Using a token without a username is valid for GitHub, but other providers (such as GitLab) require both username and tokens.</p>
+
+<p class="message--important"><strong>NOTE: </strong>If you are using a public GitHub repository, you do not need to use a secret.</p>
```

Comment:
Maybe this was on purpose, but usually the important messages have 'IMPORTANT' as a title, so they can be distinguished from the notes more easily. But just as a side note, since it is not really that important 🙈 
```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>If you are using a public GitHub repository, you do not need to use a secret.</p>
```